environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - pycrypto=2.6.1
   - pbkdf2=1.3
   - distro=1.5.0
+  - paramiko
```

Comment:
Does this need a specific version?

Reply:
yes everything needs a specific version

Reply:
The version of paramiko on the jetson nano is not available in the pip repository. I'll just put the latest version for now.